src/review/modules/components/ProductFooter.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withStyles, createStyleSheet } from 'material-ui/styles'
+import Typography from 'material-ui/Typography'
+import Divider from 'material-ui/Divider'
+import { fullWhite, darkWhite, lightWhite } from 'material-ui/styles/colors'
+import LayoutBody from 'modules/components/LayoutBody'
+import Link from 'modules/components/Link'
+import GitHub from 'review/modules/components/GitHub'
+
+const styleSheet = createStyleSheet('ProductFooter', theme => ({
+  root: {
+    background: theme.brandColor,
+    color: darkWhite,
+    overflow: 'auto',
+  },
+  title: {
+    color: fullWhite,
+  },
+  divider: {
+    margin: `${theme.spacing.unit * 2}px 0`,
+    background: 'rgba(255, 255, 255, 0.2)',
+  },
+  copyright: {
+    color: lightWhite,
+  },
+  list: {
+    margin: 0,
+    paddingLeft: 0,
+    listStyle: 'none',
+  },
+  icon: {
+    width: 16,
+    height: 16,
+    marginRight: theme.spacing.unit,
+  },
+}))
+
+function ProductFooter(props) {
+  const {
```

Comment:
1. Yes for sure.
2. Refactoring in a class is a bad smell for me, also appears in 5% of cases, VS 95% of cases. I prefer to keep it dry in 95% of cases.
3. OK

The biggest problem for me is the lose of meaning in the signature, you can't answer this question by reading the signature of the function:

- What is the type of `props`
- Is it required?
- OK I assume it's an object, what does it contains?

Destructuring could be optimized in JS engines (OK this function need an object that is required).